user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+.. _admin-capabilities:
+
+Capabilities List
+=================
+
+The *"Capabilities"*  are an important part of the system's access and permission settings, as described in the 
+:ref:`Access and Permissions Basics guide <user-guide-user-management-permissions-basic>`. 
+
+Depending on the capabilities assigned to the roles, the users will have (or have no) access to different 
+functionalities. For example, you can limit the users' abiltiy to assign tags, share grids, or create new users.
```

Comment:
Edit: Users have access to different functionalities depending on which capabilities they have been assigned. For example, you can limit the users' ability to assign tags, share grids, or create new users.